src/plugins/visualize/public/application/components/visualize_editor_common.tsx

```diff
@@ -13,12 +13,16 @@ import { EuiScreenReaderOnly } from '@elastic/eui';
 import { AppMountParameters } from 'kibana/public';
 import { VisualizeTopNav } from './visualize_top_nav';
 import { ExperimentalVisInfo } from './experimental_vis_info';
+import { DeprecationWarning } from './deprecation_vis_warning';
 import {
   SavedVisInstance,
   VisualizeAppState,
   VisualizeAppStateContainer,
   VisualizeEditorVisInstance,
 } from '../types';
+import { getUISettings } from '../../services';
+
+const LEGACY_CHARTS_LIBRARY = 'visualization:visualize:legacyChartsLibrary';
```

Comment:
I don't import it from xy plugin because it creates a cyclic dependency. As this code and setting is going to removed in the next minor, I think it is ok.